zktop.py

```diff
@@ -93,14 +95,14 @@ def __init__(self, session, server_id):
 class ZKServer(object):
     def __init__(self, server, server_id):
         self.server_id = server_id
-        self.host, self.port = server.split(':')
+        self.host, self.port = server.split(':') if ':' in server else (server, ZK_DEFAULT_PORT) # fallback to default if user doesn't specify port number
```

Comment:
Sure, will do. I like the ternary expression style very much for these simple assignments, because it makes it clear what is the normal (default) case and what the exceptional one. Also, when using "if then else" instead you have to code effectively two assignments with the chance of one branch never being called due to bugs or typos.